source/api_cc/src/DeepPotJAX.cc

```
@@ -0,0 +1,581 @@
+  // copy data
+  std::vector<T> result;
+  tensor_to_vector(result, retval, status);
+  // deallocate
+  TFE_DeleteTensorHandle(retval);
+  TFE_DeleteOp(op);
+  return result;
+}
+
+inline std::vector<std::string> get_vector_string(
+    TFE_Context* ctx,
+    const std::string func_name,
```

cppcheck

[performance] Function parameter 'func_name' should be passed by const reference.

(passedByValue)